dbt_platform_helper/templates/env/terraform-overrides/cfn.patches.yml

```diff
@@ -2,3 +2,6 @@
 - op: add
   path: /Resources/Cluster/Properties/ClusterName
   value: !Sub '${AppName}-${EnvironmentName}'
+
+- op: remove
+  path: /Resources/LogResourcePolicy
```

Comment:
How has this been tested?

Reply:
I was able to deploy my demodjango env by adding these lines to override, where previously I was getting `"Resource of type 'AWS::Logs::ResourcePolicy' with identifier '{"/properties/PolicyName":"demodjango-gabe-LogResourcePolicy"}' already exists."`